README.md

```diff
@@ -144,7 +144,7 @@ Each project adapter needs to export the main run function, in addition to some
 module.exports = {
   name: 'Template Project', // project name
   token: null,              // null, or token symbol if project has a custom token
-  category: 'assets',       // allowed values as shown on DefiPulse: 'derivatives', 'dexes', 'lending', 'payments', 'assets'
+  category: 'assets',       // allowed values as shown on DefiPulse: 'derivatives', 'dexes', 'lending', 'payments', 'assets', 'staking'
```

Comment:
Why has `staking` been added as a category? Kava is under the existing `lending` category.